youtube_dl/extractor/youtube.py

```diff
@@ -889,6 +892,14 @@ def _real_extract(self, url):
             'http' if self._downloader.params.get('prefer_insecure', False)
             else 'https')
 
+        start_time = None
+        parsed_url = compat_urllib_parse_urlparse(url)
+        for component in [parsed_url.fragment, parsed_url.query]:
+            query = compat_parse_qs(component)
+            if 't' in query:
```

Comment:
Done in https://github.com/jaimeMF/youtube-dl/commit/297a564beeb20ca8b00d94f5707532110631f409 and https://github.com/jaimeMF/youtube-dl/commit/2929fa0e79dfd3a1366e7e23eb4344bc93dd3a10